py3dtilers/Common/loa.py

```diff
@@ -0,0 +1,123 @@
+import os
+from os import listdir
+import sys
+import json
+from shapely.geometry import Point, Polygon
+from ..Common import ObjectToTile, ObjectsToTile, ObjectsToTileWithGeometry
+from ..Common import get_lod1
+
+
+def create_loa(objects_to_tile, loa_path):
```

Comment:
Define what loaD stands for and what an LOA is (point to standard document). Provide some hints on what this file is about...